ct-app/core/components/tcpudp_server.py

```
@@ -0,0 +1,73 @@
+    @formalin
+    async def listen_to_tcp_socket(self):
+
+        data = self.conn.recv(self.recv_buffer)
+        if len(data) == 0:
+            return
+        self.server_data(data)
+
+    @flagguard
+    @formalin
+    async def listen_to_udp_socket(self):
+        print(f"Doing it")
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)